docs/release-process.md

```diff
@@ -65,6 +65,8 @@ Release is happening on separate `release-<major>.<minor>` branch.
 
   For release candidate just reuse same branch and rebase it on every candidate until the actual release happens.
 
+1. Create small PR to master (!) to cut CHANGELOG. This helps to maintain new changelog on master. E.g: 
```

Comment:
What does the ending `E.g: ` stand for? Is there any missing part?

Reply:
Yes. You are too fast (((:

Reply:
Also I am just reviewing commits and filling changelog, standby :hugs: